variants/aws-ecs-1/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "aws-ecs-1"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+
+[package.metadata.build-variant]
+included-packages = [
+# core
+    "release",
+# docker
+    "docker-cli",
```

Comment:
Do we need to have the Docker CLI available?

Reply:
ECS doesn't depend on it, but it's mighty handy for debugging.  I'm using it to verify that the agent works correctly, but I'd be open to removing it if we think that's the right security choice.

Reply:
I think we'd probably want to reduce that surface area. My sense is that this is something that belongs in the Admin container. I might even suggest that we set `DOCKER_HOST` to allow the admin container to connect to the socket for breakglass debugging.

Reply:
I like that idea, though we'll have to be careful about it.  The Docker CLI in the admin container could conceivably be newer than the Docker engine on the host, leading to version mismatch issues.  (I *think* the upstream Docker project introduced version detection logic to limit the impact of this, but I'd want to test it to make sure it works the way we think it does before we depend on it.)

Reply:
@tjkirch Done